app/src/main/java/com/example/demomvp/data/source/remote/fetchJson/ParseDataWithJson.kt

```diff
@@ -0,0 +1,67 @@
+package com.example.demomvp.data.source.remote.fetchJson
+
+import com.example.demomvp.data.model.ChampionEntry
+import org.json.JSONException
+import org.json.JSONObject
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.net.HttpURLConnection
+import java.net.URL
+
+class ParseDataWithJson {
+
+    fun getJsonFromUrl(urlString: String?): String? {
+        val url = URL(urlString)
+        val httpURLConnection = url.openConnection() as HttpURLConnection
+        httpURLConnection.connectTimeout = TIME_OUT
+        httpURLConnection.readTimeout = TIME_OUT
+        httpURLConnection.requestMethod = METHOD_GET
+        httpURLConnection.doOutput = true
+        httpURLConnection.connect()
+
+        val bufferedReader = BufferedReader(InputStreamReader(url.openStream()))
+        val stringBuilder = StringBuilder()
+        var line: String?
+        while (bufferedReader.readLine().also { line = it } != null) {
+            stringBuilder.append(line)
+        }
+        bufferedReader.close()
+        httpURLConnection.disconnect()
+        return stringBuilder.toString()
+    }
+
+    fun parseJsonToData(jsonObject: JSONObject?, keyEntity: String): Any {
+        val data = mutableListOf<Any>()
+        try {
+            val jsonObjectData = jsonObject?.getJSONObject(keyEntity)
+            jsonObjectData?.keys()?.forEach {
+                val jsonObjects = jsonObjectData.getJSONObject(it)
+                val item = ParseDataWithJson().parseJsonToObject(jsonObjects, keyEntity)
+                if (data.size.equals(20)) return data
+                item?.let { data.add(it) }
+            }
+        } catch (e: JSONException) {
+            e.printStackTrace()
+        }
+        return data
+    }
+
+    fun parseJsonToObject(jsonObject: JSONObject?, keyEntity: String): Any? {
+        try {
+            jsonObject?.let {
+                when (keyEntity) {
+                    ChampionEntry.CHAMPION -> return ParseJson().championParseJson(it)
+                    else -> null
+                }
+            }
+        } catch (e: JSONException) {
+            e.printStackTrace()
+        }
+        return null
+    }
+
+    companion object {
+        private const val TIME_OUT = 15000
+        private val METHOD_GET: String? = "GET"
```

Comment:
dòng code này convert từ java sang sao ? 
```suggestion
        private const val METHOD_GET = "GET"
```